### PR TITLE
[PBF-781] Report ad title to ad screen.

### PR DESCRIPTION
**Pull request by brianfwong**
See also: https://git.corp.ooyala.com/projects/PBW/repos/mjolnir/pull-requests/201/overview

### CODE REVIEW

File: js/components/adPanel.js [hunk 1]
```diff
@@ -57,7 +57,12 @@ var AdPanel = React.createClass({
     var adTopBarItems = [];
 
     // // Ad title
-    var adTitle = this.props.currentAdsInfo.currentAdItem.name;
+    var adTitle = "Unknown";
+    if (this.props.currentAdsInfo.currentAdItem.ooyalaAds && this.props.contentTree.title) {
```
Comment (aeng7) [inline js/components/adPanel.js]: Can we add a null check for both this.props.currentAdsInfo and this.props.currentAdsInfo.currentAdItem and also this.props.contentTree?

Comment (brianfwong) [inline js/components/adPanel.js]: Something like:
if (this.props.currentAdsInfo && this.props.currentAdsInfo.currentAdItem && this.props.contentTree && this.props.currentAdsInfo.currentAdItem.ooyalaAds && this.props.contentTree.title) {
?

File: js/components/adPanel.js [hunk 2]
```diff
@@ -57,7 +57,12 @@ var AdPanel = React.createClass({
     var adTopBarItems = [];
 
     // // Ad title
-    var adTitle = this.props.currentAdsInfo.currentAdItem.name;
+    var adTitle = "Unknown";
+    if (this.props.currentAdsInfo && this.props.currentAdsInfo.currentAdItem && this.props.contentTree && this.props.currentAdsInfo.currentAdItem.ooyalaAds && this.props.contentTree.title) {
```
Comment (aeng7) [inline js/components/adPanel.js]: Can we split this up on multiple lines? This will help readability. Other than this, LGTM!

Comment (pilievOoyala) [inline js/components/adPanel.js]: I concur. Otherwise LGTM.

**Comment by brianfwong** [comment]
@aeng7 @pilievOoyala I've put the if statement onto multiple lines.

**Comment by pilievOoyala** [comment]
LGTM.  @aeng7  can you double check and merge if it look good to you.

**Comment by aeng7** [comment]
LGTM

**Pull request merged**